packages/node/src/generators/application/application.ts

```diff
@@ -77,6 +77,7 @@ function getWebpackBuildConfig(
     },
     configurations: {
       production: {
+        ...(options.docker && { generateLockfile: true }),
```

Comment:
I think lockfiles are needed regardless of docker being used or not because the deployment needs to install packages from the dist folder.

Reply:
It was only set on `docker` since this is a new param, and so is `generateLockfile`. The idea was not to change (and potentially break) existing solutions.

We plan on having `generateLockfile` on by default in v16.

Reply:
As discussed, it won't have any downside for the user to turn it on, and we can use it once we have unbundled support.